2.hello-kubernetes/README.md

```diff
@@ -4,6 +4,12 @@ This tutorial will get you up and running with Dapr in a Kubernetes cluster. We'
 
 ![Architecture Diagram](./img/Architecture_Diagram.png)
 
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A Kubernetes cluster, such as [Minikube](https://github.com/kubernetes/minikube), [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) and [GKE](https://cloud.google.com/kubernetes-engine/)
```

Comment:
Can we use this link ? - https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#setup-cluster